src/app/templates/OpenInExplorerChip/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC, useMemo } from 'react';
+
+import { ExternalLinkChip, ExternalLinkChipProps } from 'app/atoms/ExternalLinkChip';
+import { useExplorerBaseUrls } from 'lib/temple/front';
+
+import { OpenInExplorerChipSelectors } from './selectors';
+
+type Props = Omit<ExternalLinkChipProps, 'href'> & {
+  hash: string;
+  type?: keyof ReturnType<typeof useExplorerBaseUrls>;
+};
+
+export const OpenInExplorerChip: FC<Props> = ({
+  hash,
+  type = 'transaction',
+  tooltip = 'View on block explorer',
```

Comment:
Please do these points to make the default tooltip localizable:
1. Create an entry in locales for the default value or find it.
2. Use `?? t('someKey')` instead of this declaration of default value.